app/app.iml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove this file

Reply:
![img_1552](https://user-images.githubusercontent.com/32511935/46256044-cd306300-c4c2-11e8-8a7e-5f102c1ec7c3.JPG)